github/ci/prow-deploy/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits-0.93.yaml

```diff
@@ -0,0 +1,339 @@
+---
+presubmits:
+  kubevirt/cluster-network-addons-operator:
+    - name: pull-e2e-cnao-lifecycle-k8s-release-0.93
+      branches:
+        - release-0.93
+      always_run: true
+      optional: false
```

Comment:
please consider removing `optional: false` fields
this is the default value

Reply:
[DONE](https://github.com/kubevirt/project-infra/compare/acbc301b21bd2418fc9ec65d66827182448240a2..8f555593c8fb34af66e54e032c8d1135102f3cf1)